### PR TITLE
pomodoro module: Pomodoro state colors configurable.

### PR DESCRIPTION
**Pull request by egeskow**
You can set own color for each state by config string:
    colors = "on=#FFFFFF, off=#FF0000, break=#FCE94F".

Format like in keyboard_layout module

### CODE REVIEW

File: py3status/modules/pomodoro.py [hunk 1]
```diff
@@ -266,13 +270,21 @@ def pomodoro(self, i3s_output_list, i3s_config):
             response['urgent'] = True
             self._alert = False
 
+        if self.colors:
+            try:
+                self._colors = dict((k.strip(), v.strip()) for k, v in (
+                    color.split('=') for color in self.colors.split(',')))
+            except:
```
Comment (pydsigner) [inline py3status/modules/pomodoro.py]: Don't use bare exceptions please.

**Comment by egeskow** [comment]
Fixed this to catch only one error type.
User notification added. 

Should module just crash with config errors?

**Comment by tobes** [comment]
I'm not a fan of the comma delimitated string.  If #282 gets accepted then we have proper dicts etc which would be better.  Alternatively #279 effectively solves this issue.

**Comment by egeskow** [comment]
agreed with tobes

**Pull request closed without merging**